kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/QuestionListPreview.vue

```diff
@@ -82,71 +149,177 @@
 
 <script>
 
+  import { ref, computed, toRefs, watch } from 'kolibri.lib.vueCompositionApi';
   import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
-  import { displayQuestionTitle } from 'kolibri-common/strings/enhancedQuizManagementStrings';
+  import useAccordion from 'kolibri-common/components/useAccordion';
+  import AccordionItem from 'kolibri-common/components/AccordionItem';
+  import AccordionContainer from 'kolibri-common/components/AccordionContainer';
+  import coreStrings from 'kolibri.utils.coreStrings';
+  import { annotateSections } from 'kolibri.utils.exams';
```

Comment:
I appreciate this. I was thinking something along these lines earlier today but decided to just barrel ahead to get things done. If only I'd rebased earlier😭